cardano-node/src/Cardano/Node/Configuration/TopologyP2P.hs

```diff
@@ -256,6 +281,14 @@ readTopologyFile tr nc = do
     , "configuration flag. "
     , Text.pack err
     ]
+  handlerBootstrap :: Text
+  handlerBootstrap = mconcat
+    [ "You seem to have not configured any trustable peers. "
+    , "This is important in order for the node to make progress "
+    , "in bootstrap mode. Make sure you provide at least one bootstrap peer "
+    , "source (by setting 'bootstrapPeer' topology file option) "
```

Comment:
Maybe we don't need to mention how to enable bootstrap peers, since this warning is shown only if bootstrap peers are enabled.